zerver/decorator.py

```diff
@@ -335,10 +338,14 @@ def log_unsupported_webhook_event(request: HttpRequest, summary: str) -> None:
 
 def log_exception_to_webhook_logger(request: HttpRequest, err: Exception) -> None:
     extra = {"request": request}
+    # We deliberately skip logging this client error, as it results from a malformed request
+    # and doesn't indicate an issue on our end.
+    if isinstance(err, MissingHTTPEventHeaderError):
+        return
```

Comment:
Put a blank line before and after your new block, it will make this function more readable.